code/public/index.html

```diff
@@ -13,7 +13,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <title> ✅To Do App </title>
```

Comment:
As I mentioned during the group session I would highly recommend downloading an icon for the favicon instead of "hard" coding it. 

<img width="211" alt="Skärmavbild 2021-04-27 kl  20 05 14" src="https://user-images.githubusercontent.com/51661887/116290799-29031b80-a794-11eb-8567-6aac66f0b6a9.png">